client/my-sites/domains/domain-management/dns/add-dns-record.jsx

```diff
@@ -46,15 +46,19 @@ class AddDnsRecord extends Component {
 		const items = [
 			{
 				label: translate( 'Domains' ),
-				href: domainManagementList( selectedSite.slug, selectedDomainName ),
+				href: domainManagementList(
+					selectedSite?.slug,
+					selectedDomainName,
+					selectedSite?.options?.is_domain_only
+				),
 			},
 			{
 				label: selectedDomainName,
-				href: domainManagementEdit( selectedSite.slug, selectedDomainName, currentRoute ),
+				href: domainManagementEdit( selectedSite?.slug, selectedDomainName, currentRoute ),
```

Comment:
I had to change all these `selectedSite` references in various components to `selectedSite?`, otherwise when the "Domains" breadcrumb link was clicked, Calypso would break due to `selectedSite` being undefined. For some reason, the same component is rendered again when the breadcrumb link is clicked, before the "All domains" component is rendered.

If anyone knows a way to fix this, please comment 😅

Reply:
Oh, we can also pair on this one - no idea what it might be